lib/rouge/lexers/shell.rb

```diff
@@ -9,10 +9,12 @@ class Shell < RegexLexer
 
       tag 'shell'
       aliases 'bash', 'zsh', 'ksh', 'sh'
-      filenames '*.sh', '*.bash', '*.zsh', '*.ksh',
-                '.bashrc', '.zshrc', '.kshrc', '.profile', 'APKBUILD', 'PKGBUILD'
+      filenames '*.sh', '*.bash', '*.zsh', '*.ksh', '.bashrc', '.zshrc',
+                '.kshrc', '.profile', 'APKBUILD', 'PKGBUILD', '*.ebuild',
+                '*.eclass', '*.exheres-0', '*.exlib'
```

Comment:
Ugh—sorry, I must be half asleep. I should have asked for unit tests for the new file globs. Could you add these as well? These go in the same file as the unit tests for MIME types.

Reply:
done that few minutes ago

Reply:
Oops! You beat me to it! Thanks!

Reply:
https://github.com/rouge-ruby/rouge/pull/716/commits/df97237670d4903ae299c6c57d4b406dc2e017cf